DashAI/back/dataloaders/params_schemas/jsondataloader.json

```diff
@@ -1,118 +1,129 @@
+              "description": "Determines whether the data will be shuffle when defining the sets or not. It must be true for shuffle the data, otherwise false.",
+              "type": "boolean",
+              "default": true
             }
+          ]
         },
-        "more_options": {
-            "display": "modal",
-            "properties": {
-                "shuffle": {
-                    "oneOf": [
-                        {
-                            "error_msg": "Must be true or false, choose if you want to shuffle the data when separating the sets.",
-                            "description": "Determines whether the data will be shuffle when defining the sets or not. It must be true for shuffle the data, otherwise false.",
-                            "type": "boolean",
-                            "default": true
-                        }
-                    ]
-                },
-                "seed": {
-                    "oneOf": [
-                        {
-                            "error_msg": "Must be an integer greater than or equal to 0.",
-                            "description": "A seed defines a value with which the same mixture of data will always be obtained. It must be an integer greater than or equal to 0.",
-                            "type": "integer",
-                            "default": 0,
-                            "minimum": 0
-                        }
-                    ]
-                },
-                "stratify": {
-                    "oneOf": [
-                        {
-                            "error_msg": "Must be true or false, choose if you want to separate the data into sets with the same proportion of samples per class as the original set or not.",
-                            "description": "Defines whether the data will be proportionally separated according to the distribution of classes in each set.",
-                            "type": "boolean",
-                            "default": false
-                        }
-                    ]
-                }
-            },
-            "type": "object"
+        "seed": {
+          "oneOf": [
+            {
+              "error_msg": "Must be an integer greater than or equal to 0.",
+              "description": "A seed defines a value with which the same mixture of data will always be obtained. It must be an integer greater than or equal to 0.",
+              "type": "integer",
+              "default": 0,
+              "minimum": 0
+            }
+          ]
         },
-        "type": "object"
+        "stratify": {
+          "oneOf": [
+            {
+              "error_msg": "Must be true or false, choose if you want to separate the data into sets with the same proportion of samples per class as the original set or not.",
+              "description": "Defines whether the data will be proportionally separated according to the distribution of classes in each set.",
```

Comment:
"This option ensures that when dividing the dataset into train, test, and validation sets, the original distribution of labels in the input dataset will be preserved in each of the resulting datasets. This option is applicable only to classification problems"